notebooks/concepts/imviz_compass_mpl.ipynb

```diff
@@ -0,0 +1,578 @@
+    "         'CDELT1': -0.0002777777778,\n",
+    "         'CRPIX1': 1,\n",
+    "         'CRVAL1': 337.5202808,\n",
+    "         'NAXIS1': 10,\n",
+    "         'CTYPE2': 'DEC--TAN',\n",
+    "         'CUNIT2': 'deg',\n",
+    "         'CDELT2': 0.0002777777778,\n",
+    "         'CRPIX2': 1,\n",
+    "         'CRVAL2': -20.833333059999998,\n",
+    "         'NAXIS2': 10})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d2e73414",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "_ = wcs_utils.draw_compass_mpl(a, w)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a567e019",
+   "metadata": {},
+   "source": [
+    "Now, let's try that in the Compass plugin. We will display this same image in Imviz. Open the Compass plugin to see the result."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c6ee0ec6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hdu = fits.ImageHDU(a, name='SCI')\n",
+    "hdu.header.update(w.to_header())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "44b8eb24",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "imviz = Imviz()\n",
+    "imviz.load_data(hdu, data_label='arr_with_simple_wcs')\n",
+    "imviz.app"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "321b9356-abc6-4a8f-b76c-7f9bf284a248",
+   "metadata": {},
+   "source": [
+    "## Case 3: HST/ACS FLT with distortion\n",
+    "\n",
+    "We replaced the actual ACS image (`jb5g05ubq_flt.fits`) with random data to avoid data download, but the WCS is real."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "42a9e142",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a = np.random.random((2048, 4096))\n",
+    "w = WCS({'WCSAXES': 2,\n",
+    "         'CRPIX1': 2100.0,\n",
+    "         'CRPIX2': 1024.0,\n",
+    "         'PC1_1': -1.14852e-05,\n",
+    "         'PC1_2': 7.01477e-06,\n",
+    "         'PC2_1': 7.75765e-06,\n",
+    "         'PC2_2': 1.20927e-05,\n",
+    "         'CDELT1': 1.0,\n",
+    "         'CDELT2': 1.0,\n",
+    "         'CUNIT1': 'deg',\n",
+    "         'CUNIT2': 'deg',\n",
+    "         'CTYPE1': 'RA---TAN',\n",
+    "         'CTYPE2': 'DEC--TAN',\n",
+    "         'CRVAL1': 3.581704851882,\n",
+    "         'CRVAL2': -30.39197867265,\n",
+    "         'LONPOLE': 180.0,\n",
+    "         'LATPOLE': -30.39197867265,\n",
+    "         'MJDREF': 0.0,\n",
+    "         'RADESYS': 'ICRS'})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7bd46c49",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "_ = wcs_utils.draw_compass_mpl(a, w)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4862b82c",
+   "metadata": {},
+   "source": [
+    "Now, let's try that in the Compass plugin. We will display this same image in Imviz. Open the Compass plugin to see the result."
```

Comment:
| This PR | Ginga |
| --- | --- |
| ![hst_acs_flt_wcs_imviz](https://user-images.githubusercontent.com/2090236/145665071-1ad9fb6f-0dd8-404f-8144-e54902bd85fe.jpg) | ![hst_acs_flt_wcs_ginga](https://user-images.githubusercontent.com/2090236/144470727-7a85f832-b325-4b0e-a8b5-1962dad222a0.jpg) |